Source/ablastr/fields/PoissonSolver.H

```diff
@@ -17,7 +17,7 @@
 #include <ablastr/fields/MultiFabRegister.H>
 #include <ablastr/profiler/ProfilerWrapper.H>
 
-#if defined(ABLASTR_USE_FFT) && defined(WARPX_DIM_3D)
+#if (defined(ABLASTR_USE_FFT) || defined(ABLASTR_USE_HEFFTE)) && defined(WARPX_DIM_3D)
```

Comment:
Hm, now that `is_igf_3d` is a runtime option, I think that `ABLASTR_USE_FFT` is always required, correct?